app/services/feedbacks/action.rb

```diff
@@ -0,0 +1,23 @@
+require "dry-monads"
+
+module Feedbacks
+  class Action
```

Comment:
How about using `dry-transaction` for this class? https://dry-rb.org/gems/dry-transaction/main/

Reply:
Good suggestion, i will look into it